Update advanced-settings.md (#233342)
elastic/kibana: docs/reference/advanced-settings.md

$$$securitysolution-enablenewsfeed$$$`securitySolution:enableNewsFeed`
:   Enables the security news feed on the Security **Overview** page.

$$$securitySolution-enableVisualizationsInFlyout$$$`securitySolution:enableVisualizationsInFlyout`

> :   Enable visualizations (analyzer and session viewer) in document details flyout.

Reviewer: ```suggestion
:   Enables visualizations (analyzer and session viewer) in document details flyout.
```

$$$securitySolution-excludeColdAndFrozenTiersInAnalyzer$$$`securitySolution:excludeColdAndFrozenTiersInAnalyzer`

> :   When enabled, cold and frozen tiers will be skipped in analyzer queries

Reviewer: ```suggestion
:   Skips cold and frozen tiers in analyzer queries.
```

$$$securitySolution-excludedDataTiersForRuleExecution$$$`securitySolution:excludedDataTiersForRuleExecution`

> :   When configured, events from the specified data tiers are not searched during rules executions. 

Reviewer: ```suggestion
:   Excludes specified data tiers during rules executions. 
```
